### ISSUE TITLE
Temperature, humidity, new set from local devise refreshment

### ISSUE THREAD

**Issue by fredtchal** [fjhorrillo/homebridge-airzone-cloud#4]
**Describe Your Problem:**

Hello, it seems that the current (real) temperature and humidity and a new temperature set point when settled from local remote) are not refreshed on HomeKit

I Have no problem to make automation through HomeKit and the on off and set point are correctly responding 

see the log warning 
**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```
![EBD61B9D-4648-4025-A139-12E0C4D9BA1A](https://user-images.githubusercontent.com/80193749/110234047-a646a880-7f28-11eb-9cbf-be63906788ec.jpeg)

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```
As advised 
**Screenshots:**

**Environment:**

* **Plugin Version**:v0.0.2
* **Homebridge Version**: v4.39.1
* **Node.js Version**: v14.15.4
* **NPM Version**: v6.14.11
* **Operating System**: raspbian

**Comment by fjhorrillo** [comment]
Hello, this looks that something from your system is different from what I expected in the plugin. Could you provide some more info, first of all the full log with -D option to go a bit deeper over your system.

**Comment by fredtchal** [comment]
Hello thanks for the answer...
I have 4 rooms:1master +3 slaves
Main system is toshiba and split system is air-zone

I put log files in -D mode since restart of home bridge

Regards Frederic

[homebridge.log 2.txt](https://github.com/fjhorrillo/homebridge-airzone-cloud/files/6097938/homebridge.log.2.txt)

**Comment by jmartinezs77** [comment]
Hi fjhorrillo, 
Same issue here. Some more info:

- Plug in version: v0.0.2
- Sistema Operativo | Alpine Linux (3.12.4)
- Versión de Node.js: v14.16.0
- Versión de npm: v6.14.11
- Homebridge version: 1.3.4

Homebridge is running on a docker (image used: https://github.com/oznu/docker-homebridge) within a Synology NAS.

I am also having the issue of temperatures not refreshing if zones that are turned off.

Many thanks for this great plugin, never thought i could get airzone in homekit.

Saludos

PD: can send logs, but would need help to turn on debugging, not sure the value to provide on the debug field of homebridge.

**Comment by fjhorrillo** [comment]
Fixed in version 0.1.1
